geopandas/base.py

```diff
@@ -498,6 +498,23 @@ def intersection(self, other):
         """
         return _binary_geo("intersection", self, other)
 
+    def covers(self, other):
+        """
+        Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        each geometry that is entirely covering `other`.
+
+        An object A is said to cover another object B if no points of B lie
+        in the exterior of A.
+
+        See `linearthinking`_ for reference.
```

Comment:
You can put the link directly here (if you put the link on a single line, I think the linter should not complain, otherwise we can split the link over two lines with `\`, which will still look as a single line when looking at the docstring)